test/e2e/bundle_e2e_test.go

```diff
@@ -47,6 +118,30 @@ var _ = Describe("Installing bundles with new object types", func() {
 				imageName  = "quay.io/olmtest/single-bundle-index:pdb"
 			)
 
+			// create VPA CRD on cluster
+			data, err := yaml.YAMLToJSON([]byte(vpaCRDManifest))
+			Expect(err).ToNot(HaveOccurred(), "could not convert vpa crd to json")
+
+			crd := &apiextensionsv1beta1.CustomResourceDefinition{}
+			err = json.Unmarshal(data, crd)
+			Expect(err).ToNot(HaveOccurred(), "could not convert vpa crd to unstructured")
+
+			_, err = kubeClient.ApiextensionsInterface().ApiextensionsV1beta1().CustomResourceDefinitions().Create(context.TODO(), crd, metav1.CreateOptions{})
+			if err != nil {
+				if !k8serrors.IsAlreadyExists(err) {
+					Expect(err).ToNot(HaveOccurred(), "could not create vpa crd to on cluster")
+				}
+			}
+
+			// ensure vpa crd is established and accepted on the cluster before continuing
```

Comment:
Are we just masking an issue with `InstallPlan` reconciliation? e.g. `InstallPlans` should eventually install CRs once a CRD becomes available.

Reply:
They should but I believe we currently have an issue where the user provides the VPA CRD in the same bundle as the VPA CR. Although I guess in practice this won't be the case? Maybe this is just an issue with the way the test was written originally. I'm not sure having this check hurts though.

Reply:
It does result in using the `v1beta1` client to check the status, but I don't think its a big deal. When we move off `v1beta1` we will require a lot of updates to our e2e suite.